modules/users/tests/server/user.server.model.tests.js

```diff
@@ -71,7 +70,6 @@ describe('User Model Unit Tests:', function () {
     it('should fail to save an existing user again', function (done) {
       var _user = new User(user);
       var _user2 = new User(user2);
-
```

Comment:
Don't we want the space here to separate variable declarations for the test from the actual execution?